mount.go

```diff
@@ -26,6 +27,50 @@ const (
 	mountPerm      = os.FileMode(0755)
 )
 
+var flagList = map[string]int{
+	"acl":           unix.MS_POSIXACL,
+	"async":         unix.MS_SYNCHRONOUS,
+	"atime":         unix.MS_NOATIME,
+	"bind":          unix.MS_BIND,
+	"defaults":      0,
+	"dev":           unix.MS_NODEV,
+	"diratime":      unix.MS_NODIRATIME,
+	"dirsync":       unix.MS_DIRSYNC,
+	"exec":          unix.MS_NOEXEC,
+	"iversion":      unix.MS_I_VERSION,
+	"lazytime":      unix.MS_LAZYTIME,
+	"loud":          unix.MS_SILENT,
+	"mand":          unix.MS_MANDLOCK,
+	"noacl":         unix.MS_POSIXACL,
+	"noatime":       unix.MS_NOATIME,
+	"nodev":         unix.MS_NODEV,
+	"nodiratime":    unix.MS_NODIRATIME,
+	"noexec":        unix.MS_NOEXEC,
+	"noiversion":    unix.MS_I_VERSION,
+	"nolazytime":    unix.MS_LAZYTIME,
+	"nomand":        unix.MS_MANDLOCK,
+	"norelatime":    unix.MS_RELATIME,
```

Comment:
The handling for "noiversion", "nolazytime", "nomand", "norelatime" ... "suid" are incorrect.
The corresponding flags should be cleared rather than be added.

Reply:
This looks to be a re-implementation of code already in libcontainer as used by the agent. Unfortunately, the function is private :sad::

- https://github.com/opencontainers/runc/blob/master/libcontainer/specconv/spec_linux.go#L646..L702

Reply:
@laijs oh yeah you're right and actually we don't need those `no...` flags.

Reply:
"noexec", "nodev", "nosuid" are removed in your updated pr.
I think it's better to copy the exact code.

Reply:
I have updated the PR by making sure I was adding only what matches a flag. I don't think we need to copy the entire code from https://github.com/opencontainers/runc/blob/master/libcontainer/specconv/spec_linux.go#L648-L732 since I don't expect a user of the gRPC protocol using `dev` for example in case he wants `nodev` to be disabled. The caller should only ask for flags to be enabled since it controls the entire list of flags.